examples/basic/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,23 +1,13 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <uses-feature
```

Comment:
can you please revert changes in this file to keep "current" android tv support ?